independent-publisher-connectors/Cisco Webex List Meeting/apiDefinition.swagger.json

```diff
@@ -0,0 +1,197 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cisco Webex List Meetings",
+    "description": "The meeting API provides meeting details. The detail includes the client detail, meeting time, and meeting duration.",
+    "version": "1.0",
+    "contact": {
+      "name": "Woong Choi",
+      "url": "https://www.linkedin.com/in/woongchoi/"
+    }
+  },
+  "host": "webexapis.com",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/meetings": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "items": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Unique identifier for meeting. For a meeting series, the id is used to identify the entire series. For scheduled meetings from a series, the id is used to identify that scheduled meeting. For a meeting instance that is in progress or has concluded, the id is used to identify that instance."
+                      },
+                      "meetingSeriesId": {
+                        "type": "string",
+                        "description": "Meeting number. Applies to meeting series, scheduled meeting, and meeting instances, but not to meeting instances which have ended."
+                      },
+                      "scheduledMeetingId": {
+                        "type": "string",
+                        "description": "Id for Instance from a primary meeting series."
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "Meeting title. Can be modified for a meeting series or a scheduled meeting using the Update a Meeting API."
+                      },
+                      "meetingType": {
+                        "type": "string",
+                        "description": "Meeting type. meetingSeries: Primary instance of a scheduled series of meetings which consists of one or more scheduled meetings based on a recurrence rule. When a non-recurring meeting is scheduled with no recurrence, its meetingType is also meetingSeries which is a meeting series with only one occurrence in Webex meeting modeling.; scheduledMeeting: Instance from a primary meeting series.; meeting: Meeting instance that is in progress or has completed."
+                      },
+                      "state": {
+                        "type": "string",
```

Comment:
@last72 I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).